src/cbottle/inference.py

```diff
@@ -808,7 +831,13 @@ def __init__(
 
     @classmethod
     def from_pretrained(
-        cls, path: str | list[str], sigma_thresholds: tuple[float, ...]
+        cls,
+        path: str | list[str],
+        sigma_thresholds: tuple[float, ...],
+        *,
+        use_apex_groupnorm: bool | None = None,
```

Comment:
None and bool should not be combined for a flag

```suggestion
        use_apex_groupnorm: bool = False,
```